Sources/Benchmark/BenchmarkArguments.swift

```diff
@@ -39,6 +39,12 @@ public struct BenchmarkArguments: ParsableArguments {
     @Option(help: "Time unit used to report the results.")
     var timeUnit: TimeUnit.Value?
 
+    @Option(help: "Time unit used to report the results.")
+    var inverseTimeUnit: TimeUnit.Value?
```

Comment:
Could you please add a doc comment explaining the purpose of this? (And perhaps also update the help text too.)

Reply:
Fixed in fa352a9